src/inmanta/server/services/databaseservice.py

```diff
@@ -118,7 +119,7 @@ def get_pool_free(self) -> int:
             return 0
         return self._pool.get_max_size() - self._pool.get_size() + self._pool.get_idle_size()
 
-    def _add_gauge(self, name: str, the_gauge: CallbackGauge) -> None:
+    def _add_gauge(self, name: str, the_gauge: AnyGuage) -> None:
```

Comment:
Could even make it generic here as well?
```suggestion
    def _add_gauge(self, name: str, the_gauge: Gauge[T]) -> None:
```